packages/igx-templates/igx-ts/dock-manager/default/index.ts

```diff
@@ -0,0 +1,23 @@
+import { TypeScriptFileUpdate } from "@igniteui/cli-core";
+import { IgniteUIForAngularTemplate } from "../../../IgniteUIForAngularTemplate";
+
+class IgxDockManagerTemplate extends IgniteUIForAngularTemplate {
+	constructor() {
+		super(__dirname);
+		this.components = ["Dock Manager"];
+		this.controlGroup = "Layouts";
+		this.listInComponentTemplates = true;
+		this.id = "dock-manager";
+		this.projectType = "igx-ts";
+		this.name = "Dock Manager";
+		this.description = "basic IgxDockManager";
```

Comment:
```suggestion
		this.description = "basic IgcDockManager";
```
Not sure that's the correct name, but still it's not `Igx` for sure